scripts/local-dev.sh

```diff
@@ -65,12 +65,9 @@ function up() {
         mk start \
             --driver="${DRIVER}" \
             --nodes="${NODE_COUNT}" \
-            --kubernetes-version=v1.23.3 \
             --cpus=2 \
             --memory=8gb \
-            --disk-size=20gb \
-            --docker-opt dns=8.8.8.8 \
```

Comment:
Have we checked whether this config runs smoothly with Tilt locally? Like collecting metrics and visualizing them with Parca Server? All these are added for a reason, maybe things has changed, so let's make sure. 

This could also be because of using different drivers. Let's check with KVM and VirtualBox and make sure everything works. I can do these as well.

Reply:
I've never tried Tilt, but I've used half of the drivers and never faced DNS issues with Minikube. I'm ok with reverting these changes to shorten the discussion since this script is a little out of the scope of the PR, just trying keep it close to the CI